lib/utils/ember.js

```diff
@@ -218,16 +219,27 @@ function isRoute(node) {
     node.callee.property.name === 'route';
 }
 
-function isRouteMethod(name) {
+function isRouteLifecycleHookName(name) {
   return [
     'activate',
-    'addObserver',
     'afterModel',
     'beforeModel',
+    'deactivate',
+    'model',
+    'redirect',
+    'renderTemplate',
+    'resetController',
+    'serialize',
+    'setupController'
+    ].indexOf(name) > -1;
+}
+
+function isRouteMethod(name) {
```

Comment:
I think we can actually remove this method.

https://github.com/rwwagner90/eslint-plugin-ember/blob/8c0cfd7c6add13169c0df5882a4bfafcca6c6499/lib/utils/ember.js#L284

We should replace in above code `isRouteMethod` with `isRouteLifecycleHookName` and additionally remove `isRouteDefaultMethod` too.